main.py

```diff
@@ -149,11 +151,16 @@ def file_paths_to_dict(resume_file: Path | None, plain_text_resume_file: Path) -
 
         return result
 
-def init_browser() -> webdriver.Chrome:
+def init_browser(browser: str = 'chrome') -> webdriver.Chrome | webdriver.Firefox:
```

Comment:
use abstractions or interface for web driver where base driver can point to chrome driver or firefox dirver (polymorphism)

Reply:
I’ve abstracted the browser type here. For the return type, I prefer to address it in a separate PR since it will require refactoring other `aihawk*` files as well.

Reply:
Additionally, if the webdriver parts need refactoring, I would recommend using [Playwright](https://playwright.dev/) since it natively supports multiple browsers.

Reply:
we might need to evaluate playwright & its compatibility with existing. but you create a feature request for it.

Reply:
> I’ve abstracted the browser type here. For the return type, I prefer to address it in a separate PR since it will require refactoring other `aihawk*` files as well.

As long as this PR runs smoothly, we are happy to merge, No problem creating a separate PR for return type. do map that PR with this PR so reviewers can understand.